tests/FakeItEasy.Tests/Configuration/FakeConfigurationManagerTests.cs

```diff
@@ -147,18 +164,21 @@ public void Should_return_configuration_when_configuring_any_call()
             result.Should().BeSameAs(expectedConfiguration);
         }
 
-        [Fact]
-        public void Should_add_call_rule_to_fake_manager_when_configuring_any_call()
+        [SuppressMessage("Microsoft.Design", "CA1062:Validate arguments of public methods", MessageId = "0", Justification = "It's not really a public method")]
+        [Theory]
+        [MemberData(nameof(CallSpecificationActions))]
+        public void CallTo_should_not_add_rule_to_manager(Action<FakeConfigurationManagerTests, IFoo> action)
         {
             // Arrange
-            var fake = A.Fake<IFoo>();
-            var manager = Fake.GetFakeManager(fake);
+            var foo = A.Fake<IFoo>();
+            var manager = Fake.GetFakeManager(foo);
+            var initialRules = manager.Rules.ToList();
 
             // Act
-            this.configurationManager.CallTo(fake);
+            action(this, foo);

```

Comment:
I found all the `@this`es up near line 35 distracting. How do you feel about just passing `configurationManager` in here instead of `this`? I think it better reveals which object's being manipulated.